content/400-reference/300-database-reference/01-database-features.mdx

```diff
@@ -123,18 +123,18 @@ This section describes which database features exist on the NoSQL databases that
+| :---------------------------------------- | :------------------ | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Embedded documents                        | ✔️                  |                                                                                                                                                                                              |
+| Transactions                              | ✔️                  |                                                                                                                                                                                              |
+| Indexes                                   | ✔️ with caveats     | Indexes can only be introspected if the field they refer to includes at least some data.                                                                                                     |
+| Autoincrementing IDs                      | No                  |                                                                                                                                                                                              |
+| Compound IDs                              | No                  | MongoDB does not support composite IDs (`@@id`)                                                                                                                                              |
+| Generated `ObjectId`                      | ✔️                  | See: [Defining IDs for MongoDB](/concepts/components/prisma-schema/data-model#defining-ids-in-mongodb)                                                                                       |
+| Arrays                                    | ✔️                  |                                                                                                                                                                                              |
+| Enums                                     | ✔️                  | Implemented at Prisma level                                                                                                                                                                  |
+| Native database types                     | ✔️                  | See: [Field mapping reference](/reference/api-reference/prisma-schema-reference#model-field-scalar-types)                                                                                    |
+| JSON support                              | ✔️                  | Advanced `Json` field filtering is supported by [Postgres and MySQL since 2.26.0](/concepts/components/prisma-client/working-with-fields/working-with-json-fields#filtering-on-a-json-field) |
```

Comment:
Why does the PostgreSQL and MySQL support matter in a table about MongoDB?

Reply:
Excellent question, I missed that. Does MongoDB support `Json` field?

Reply:
Yes. Not sure about the filtering etc though. @dpetrick ?

Reply:
@matthewmueller can you answer above question, about whether or not MongoDB supports `Jason` field?

Reply:
got the answer in slack thanks